src/main/java/com/endava/soa_5ed/task5_io/ex1/PropertiesManager.java

```diff
@@ -0,0 +1,43 @@
+package com.endava.soa_5ed.task5_io.ex1;
+
+import java.io.*;
+import java.util.Properties;
+
+public class PropertiesManager {
+    String result;
+    private Properties properties;
+
+    public PropertiesManager(String filepath) {
+        try {
+            properties = readPropFile(filepath);
+        } catch (IOException ioException) {
+            ioException.printStackTrace();
+        }
+    }
+
+    public Properties readPropFile(String filepath) throws IOException {
+        FileInputStream inputStream = null;
+        Properties prop = null;
+        try {
+            inputStream = new FileInputStream(filepath);
+            prop = new Properties();
+            prop.load(inputStream);
+        } catch (FileNotFoundException fileNotFound) {
```

Comment:
Why do we need both catch-es ?

Reply:
because of the inputStream.close() the readPropFile() throws an IO Exception so we're handling it in the constructor

Reply:
I deleted the second catch from readPropFile() 
thank you